src/css/admin.css

```
@@ -481,4 +481,45 @@ fieldset.tinypng_convert_fields[disabled] {
+	border: 1px solid #e1e1e1;
+}
+
+.tiny-card a.button {
+	display: inline-flex;
+	align-items: center;
+	gap: 4px;
+}
+
+.tiny-icon-backup {
+	flex-shrink: 0;
+	fill: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)